### DISCUSSION TITLE
Log or dump curl-errors

### DISCUSSION THREAD

**Discussion by iammati** [spatie/ray#276] [Ideas]
Hey there,

I was about to try out this new debugging application. When trying to use the simple ray-function nothing happened.
I had to debug the composer-package (spatie/ray) on a TYPO3 (PHP) project by adding a dd (symfony/var-dumper) to see what the curl actually throws.

![image](https://user-images.githubusercontent.com/41418763/106797937-9011a800-665d-11eb-9b58-45fc48f72b71.png)
Even though such things might be easy for server administrators who are more or less familiar with the Connection refused, I'd suggest to throw exceptions on a development-based projects to see which things are not running properly.

Would like to hear your opinion too!

Greets

**Comment by freekmurze** [comment]
We don't throw exceptions so people's projects do not blow up when Ray is not running

**Reply by iammati** [reply]
How about a configurable way which is by default to false to throw them? Or to log them at least to notice something..?
